Editor/AGS.Editor/Components/AudioComponent.cs

```diff
@@ -323,25 +325,67 @@ private void ImportAudioFiles(string[] selectedFiles)
             }
         }
 
+        private AudioClip SetAudioClipSourceFile(AudioClip clip, string fullSourceFileName)
+        {
+            if (string.IsNullOrEmpty(fullSourceFileName))
+            {
+                return null;
+            }
+
+            if(!File.Exists(fullSourceFileName))
+            {
+                throw new FileNotFoundException("Source file doesn't exist.");
+            }
+            else
+            {
+                byte[] audioData = File.ReadAllBytes(fullSourceFileName);
+                if (audioData.Length == 0) throw new FileLoadException("New source file is empty.");

```

Comment:
I found there's a FileInfo class that can let know the length of the file without reading its data:
https://learn.microsoft.com/en-us/dotnet/api/system.io.fileinfo?view=netframework-4.6

Reply:
Thanks, I added this in [the push below](https://github.com/adventuregamestudio/ags/compare/9a683d6b52f950ce9b2b3bca36b61a21db95b0c7..daff42c9256d63d3a45e9ae27b0603c560d66be3)